charts/actions-runner-controller-2/values.yaml

```diff
@@ -62,4 +62,9 @@ affinity: {}
 # Leverage a PriorityClass to ensure your pods survive resource shortages
 # ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
 # PriorityClass: system-cluster-critical
-priorityClassName: ""
\ No newline at end of file
+priorityClassName: ""
+
+logging:
```

Comment:
To anticipate other changes to the manager, maybe we should introduce a top level struct like following:
```yaml
manager:
  flags:
    logLevel: "debug"
```
What do you think? :relaxed: